### ISSUE TITLE
Missing torrc config

### ISSUE THREAD

**Issue by nukeador** [HiroshiManRise/anonym8#2]
The script removed all existing tor config with no backup. If the config is not working on the system is not possible to go back once you disable the script

**Comment by HiroshiManRise** [comment]
Right, I'll add a back-up line on next version, thx for the report.
